cmake/O2Dependencies.cmake

```diff
@@ -673,3 +673,37 @@ o2_define_bucket(
 
 )
 
+o2_define_bucket(
+    NAME
+    emcal_base_bucket
+
+    DEPENDENCIES
+    root_base_bucket
+    fairroot_base_bucket
+    ParBase
+    
+    INCLUDE_DIRECTORIES
+    ${FAIRROOT_INCLUDE_DIR}
+)
+
+o2_define_bucket(
+    NAME
+    emcal_simulation_bucket
+
+    DEPENDENCIES
+    emcal_base_bucket
+    root_base_bucket
+    fairroot_geom
+    RIO
+    Graf
```

Comment:
I am wondering whether we can repackage these things into the root_bucket? It gets a configuration nightmare otherwise. Also the libnames Graf, RIO do not carry semantic information that they come from ROOT.